static/js/stream_edit.js

```diff
@@ -77,8 +77,8 @@ exports.open_edit_panel_for_row = function (stream_row) {
     clear_edit_panel();
     subs.show_subs_pane.settings();
     $(stream_row).addClass("active");
-    stream_edit.show_settings_for(stream_row);
     setup_subscriptions_stream_hash(sub);
+    stream_edit.show_settings_for(stream_row);
```

Comment:
Our UI rendering depends on function `is_subs_open` which returns false if hash is not changed. So we should first change the hash and then render UI elements.

Reply:
This would probably be cleaner if you just put it in the commit message for the commit that made the change, since then it's there for future developers debugging using `git log -p`.